rule.php

```diff
@@ -378,6 +378,32 @@ public function prevent_access() {
         }
     }
 
+    /**
+     * Hepler function to display an Exit Safe Exam Browser button if configured to do so and attempts are > 0.
+     *
+     * @return string empty or a button which has the configured seb quit link.
+     */
+    private function display_quit_button() : string {
+        global $PAGE, $USER;
+
+        $output = $PAGE->get_renderer('mod_quiz');
+        $quizsettings = $this->accessmanager->get_quiz_settings();
+        $attempts = quiz_get_user_attempts($quizsettings->get('quizid'), $USER->id, 'finished', false);
+        $quitbutton = '';
+
+        if (empty($attempts)) {
+            return $quitbutton;
+        }
+
+        // Only display if the link has been configured and attempts are greater than 0.
+        if ($quizsettings->get('linkquitseb')) {
+            $url = new moodle_url($quizsettings->get('linkquitseb'));
+            $quitbutton = $output->start_attempt_button(get_string('exitsebbutton', 'quizaccess_seb'), $url);
```

Comment:
I'd be really careful using existing start_attempt_button. Why not render a normal button?

Reply:
This now calls the standard $OUTPUT->single_button